unit-test-app/examples/unit-test-app/tools/ElfUnitTestParser.py

```diff
@@ -0,0 +1,83 @@
+# SPDX-FileCopyrightText: 2022 Espressif Systems (Shanghai) CO LTD
+# SPDX-License-Identifier: Apache-2.0
+import argparse
+import os
+import subprocess
+import sys
+from typing import Dict, List
+
+import yaml
+
+try:
+    import CreateSectionTable
+except ImportError:
+    sys.path.append(os.path.expandvars(os.path.join('$IDF_PATH', 'tools', 'unit-test-app', 'tools')))
+    import CreateSectionTable
+
+
+def get_target_objdump(idf_target: str) -> str:
+    toolchain_for_target = {
+        'esp32': 'xtensa-esp32-elf-',
+        'esp32s2': 'xtensa-esp32s2-elf-',
+        'esp32s3': 'xtensa-esp32s3-elf-',
+        'esp32c2': 'riscv32-esp-elf-',
+        'esp32c3': 'riscv32-esp-elf-',
```

Comment:
what about other targets?

Reply:
Seeing it is a legacy app and we dont really recommend using it anymore, Im not sure it make sense to update it for newer targets. (what you see here is simply whjat was in IDF)

Reply:
But I guess we can be nice since this is a simple change, changed the default dict value to riscv toolchain